edgelet/build/linux/build-k8s.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+###############################################################################
+# This script builds the project
+###############################################################################
+
+set -e
+
+###############################################################################
+# Define Environment Variables
+###############################################################################
+# Get directory of running script
+DIR=$(cd "$(dirname "$0")" && pwd)
+
+BUILD_REPOSITORY_LOCALPATH=${BUILD_REPOSITORY_LOCALPATH:-$DIR/../../..}
+PROJECT_ROOT=${BUILD_REPOSITORY_LOCALPATH}/edgelet
+IOTEDGED_MANIFEST=${PROJECT_ROOT}/iotedged/Cargo.toml
+SCRIPT_NAME=$(basename "$0")
+CARGO="${CARGO_HOME:-"$HOME/.cargo"}/bin/cargo"
+TOOLCHAIN="stable-x86_64-unknown-linux-gnu"
+RELEASE=
+
+###############################################################################
+# Print usage information pertaining to this script and exit
+###############################################################################
+usage()
+{
+    echo "$SCRIPT_NAME [options]"
+    echo ""
+    echo "options"
+    echo " -h, --help          Print this help and exit."
+    echo " -t, --toolchain     Toolchain (default: stable-x86_64-unknown-linux-gnu)"
+    echo " -r, --release       Release build? (flag, default: false)"
+    exit 1;
+}
+
+###############################################################################
+# Obtain and validate the options supported by this script
+###############################################################################
+process_args()
+{
+    save_next_arg=0
+    for arg in "$@"
+    do
+        if [ $save_next_arg -eq 1 ]; then
+            TOOLCHAIN="$arg"
+            save_next_arg=0
+        elif [ $save_next_arg -eq 2 ]; then
+            RELEASE="true"
+            save_next_arg=0
+        else
+            case "$arg" in
+                "-h" | "--help" ) usage;;
+                "-t" | "--toolchain" ) save_next_arg=1;;
+                "-r" | "--release" ) save_next_arg=2;;
+                * ) usage;;
+            esac
+        fi
+    done
+}
+
+process_args "$@"
+
+if [[ -z ${RELEASE} ]]; then
+    cd "$PROJECT_ROOT" && $CARGO "+$TOOLCHAIN" build --manifest-path=${IOTEDGED_MANIFEST} --no-default-features --features runtime-kubernetes
```

Comment:
Just do `-p iotedged` instead of `--manifest-path`

Reply:
I would but it doesn't work:
```sh
~/iotedge/edgelet$ build/linux/build-k8s.sh
    Finished dev [unoptimized + debuginfo] target(s) in 0.10s
~/iotedge/edgelet$ target/debug/iotedged -c ~/config.yaml 
<6>2019-08-23T15:18:31Z [INFO] - Starting Azure IoT Edge Security Daemon - Kubernetes mode
<6>2019-08-23T15:18:31Z [INFO] - Version - 1.0.10~dev
<6>2019-08-23T15:18:31Z [INFO] - Using config file: /home/darobs/config.yaml
<3>2019-08-23T15:18:31Z [ERR!] - The daemon could not start up successfully: Could not load settings
<3>2019-08-23T15:18:31Z [ERR!] - 	caused by: Config parsing error
<3>2019-08-23T15:18:31Z [ERR!] - 	caused by: missing field `proxy_trust_bundle_path`
~/iotedge/edgelet$ cargo build -p iotedged --no-default-features --features runtime-kubernetes 
    Finished dev [unoptimized + debuginfo] target(s) in 0.12s
~/iotedge/edgelet$ target/debug/iotedged -c ~/config.yaml 
<6>2019-08-23T15:18:47Z [INFO] - Starting Azure IoT Edge Security Daemon
<6>2019-08-23T15:18:47Z [INFO] - Version - 1.0.10~dev
```

See: https://github.com/rust-lang/cargo/issues/5015